ibis/backends/base/sql/alchemy/client.py

```diff
@@ -329,3 +329,120 @@ def _sqla_table_to_expr(self, table):
     def version(self):
         vstring = '.'.join(map(str, self.con.dialect.server_version_info))
         return parse_version(vstring)
+
+    def insert(
+        self,
+        to_table_name: str,
+        database: Optional[str] = None,
+        data_obj=None,
+        from_table_name: Optional[str] = None,
+        if_exists: Optional[str] = 'append',
+    ) -> None:
+        """
+        Insert the given data to a table in backend.
+
+        Parameters
+        ----------
+        to_table_name : string
+            name of the table to which data needs to be inserted
+        database : string, optional
+            name of the attached database that the table is located in.
+        data_obj : pd.Dataframe
```

Comment:
this is optional?

Reply:
Yes, but one of the optional parameters, either `data` or `from_table_name` must be sent by the user otherwise the function throws an error.

Reply:
Added `optional` to docstring and signature.